### DISCUSSION TITLE
Suggestions for getting around CAPTCHA

### DISCUSSION THREAD

**Discussion by kenetik** [ArchiveBox/ArchiveBox#731] [Q&A]
Any suggestions for getting around the CAPTCHA requirements on some sites?

**Comment by pirate** [comment]
Have you tried setting your CHROME_DATA_DIR to the directory containing your Chrome profile? Log in using chrome and bypass the captcha (so that the cookies are saved in your profile), then pass that profile dir to archivebox to use it for archiving.
